README.md

```diff
@@ -1072,6 +1092,20 @@ Some notes on jar packaging:
    ```
 
 ## The Logging library
```

Comment:
Should mention 2 additional things: 
* That when you `getLogger(some.class)` you can configure it explicitly in the log4j2.xml by defining `<logger>` of the same name. 
* About the fallback mechanism to root logger.

Reply:
Thank you @yudhistira-arya, I have added the details to the documentation for the SDK Logging section :)